crates/signal/src/bootstrap.rs

```diff
@@ -13,10 +13,10 @@ pub trait SignalConfig: Global {
     /// The signals to listen for.
     ///
     /// By default, listens for `SIGTERM` and `SIGINT`.
-    fn signals(&self) -> Vec<tokio::signal::unix::SignalKind> {
+    fn signals(&self) -> Vec<crate::SignalKind> {
         vec![
-            tokio::signal::unix::SignalKind::terminate(),
-            tokio::signal::unix::SignalKind::interrupt(),
+            crate::SignalKind::Unix(tokio::signal::unix::SignalKind::terminate()),
+            crate::SignalKind::Unix(tokio::signal::unix::SignalKind::interrupt()),
```

Comment:
We should change these to be the crate::SignalKind::Interupt / Terminate ones which are generic across os.